packages/dev/core/src/Maths/math.path.ts

```diff
@@ -1,6 +1,6 @@
 import type { DeepImmutable, Nullable } from "../types";
 import { Scalar } from "./math.scalar";
-import { Vector2, Vector3, Quaternion, Matrix } from "./math.vector";
+import { Vector2, Vector3, type Vector4, Quaternion, Matrix } from "./math.vector";

```

Comment:
Supporting all typescript versions

```suggestion
import { Vector2, Vector3, Quaternion, Matrix } from "./math.vector";
import type { Vector4 } from "./math.vector";
```